lib/config/folderconfiguration.go

```diff
@@ -73,9 +73,13 @@ func (f FolderConfiguration) Path() string {
 	return f.cachedPath
 }
 
+func (f FolderConfiguration) SyncthingPath() string {
+	return filepath.Join(f.Path(), ".syncthing")
+}
+
 func (f *FolderConfiguration) CreateMarker() error {
 	if !f.HasMarker() {
-		marker := filepath.Join(f.Path(), ".stfolder")
+		marker := f.SyncthingPath()
```

Comment:
The "marker" (now folder) gets hidden. Is this desirable, if we store versions and stuff in there? On Unix I think this is mostly expected and people are used to dotfiles being only somewhat hidden - is this the case on Windows too?